src/components/Image.js

```diff
@@ -1,10 +1,11 @@
-import React from "react";
+import React from 'react'
```

Comment:
@pasternakewa You ahve strange formatting sometimes single quote sometimes double. I think You formatter is off